src/main/java/org/apache/sysds/parser/BuiltinFunctionExpression.java

```diff
@@ -752,6 +752,31 @@ else if(((ConstIdentifier) getThirdExpr().getOutput())
 				raiseValidateError("Compress/DeCompress instruction not allowed in dml script");
 			break;
 
+			case QUANTIZE_COMPRESS: // this is not used when with two arguments it seems
```

Comment:
if it does not hit here, remove, you will see shortly (if all tests pass) a code coverage report, that will indicate if this is used.